smithy-model/src/test/resources/software/amazon/smithy/model/selector/enums.smithy

```diff
@@ -0,0 +1,15 @@
+$version: "2.0"
```

Comment:
Since the test is using an `enum` trait, it would be more straightforward to use an enum trait in the model rather than rely on the synthetically created enum trait.

Reply:
Updated it enum trait